googledatastore/README.md

```diff
@@ -20,6 +20,9 @@ LICENSE file.
 
 https://cloud.google.com/datastore/docs/concepts/overview?hl=en
 
+Please refer to https://cloud.google.com/datastore/docs/apis/overview for more information on
```

Comment:
Consider replace the link above with this new link here. And doing that, you can remove this "please refer to..." paragraph.